plot_krun_results.py

```diff
@@ -435,6 +447,10 @@ def create_cli_parser():
                         help=('Only draw charts for a specific benchmark/' +
                               'vm/variant triplet. ' +
                               'e.g. binarytrees:Hotspot:default-java'))
+    parser.add_argument('--zoom', '-z',
+                        help=("Zoom in on sample range "
+                              "(comma separated pair 'start,end'. Samples "
+                              "start from 0, upper bound is exclusive)"))
     return parser
```

Comment:
I think it's worth giving an example here. e.g. `Zoom in on a small sample of the measurements. e.g. '--zoom 100,250' displays measurements in the range x in [100, 250).` or some such.